app/models/organization.rb

```diff
@@ -36,11 +35,13 @@ def self.counties
     Organization.select(:county).map(&:county).uniq
   end
 
-  private def fetch_geocoding_data
+  private
+
+  def fetch_geocoding_data
     begin
-      result = Geocoder.search(address).first
+      result = Geocoder.search(addresses.first).first
```

Comment:
According to the docs I read this is the code generated by first:
`Person.first # returns the first object fetched by SELECT * FROM people ORDER BY people.id LIMIT 1`
it sorts by id automagically.